pkgs/applications/networking/expressvpn/default.nix

```diff
@@ -0,0 +1,102 @@
+{ autoPatchelfHook
+, buildFHSUserEnv
+, dpkg
+, fetchurl
+, inotify-tools
+, lib
+, stdenv
+, sysctl
+, writeScript
+}:
+
+let
+  name = "expressvpn";
+  version = "3.24.0.2-1";
+
+  expressvpnBase = stdenv.mkDerivation {
+    inherit name version;
+
+    src = fetchurl {
+      url = "https://www.expressvpn.works/clients/linux/expressvpn_${version}_amd64.deb";
+      hash = "sha256-YUbUnSYXxSPhjAvyxEfoeSxo4Zy1c4rcGgDqLrS39Wg=";
+    };
+
+    nativeBuildInputs = [ dpkg autoPatchelfHook ];
+
+    dontConfigure = true;
+    dontBuild = true;
+
+    unpackPhase = ''
+      runHook preUnpack
+      dpkg --fsys-tarfile $src | tar --extract
+      runHook postUnpack
+    '';
+
+    installPhase = ''
+      runHook preInstall
+      mv usr/ $out/
+      mv lib/systemd/ $out/lib/
+      runHook postInstall
+    '';
+  };
+
+  expressvpndFHS = buildFHSUserEnv {
+    name = "expressvpnd";
```

Comment:
Not entirely sure, don't you want to suffix this with the version as well?

Reply:
The other nixpkgs I've seen don't seem to suffix the version in FHSUserEnv so I didn't either. Is it needed?

Reply:
buildFHSUserEnv does not take pname+version.